pkg/3scale/amp/operator/apicast_reconciler.go

```diff
@@ -186,6 +186,11 @@ func (r *ApicastReconciler) Reconcile() (reconcile.Result, error) {
 		return reconcile.Result{}, err
 	}
 
+	err = r.ReconcilePrometheusRules(apicast.ApicastPrometheusRules(), reconcilers.RemovePrometheusRulesMutator)
```

Comment:
calling twice `ReconcilePrometheusRules` with the same rules and different mutator? that does not seem to be correct.

Reply:
amended. single call with new mutator